src/platform/datapath_epoll.c

```diff
@@ -2380,16 +2380,22 @@ CxPlatSendDataFreeBuffer(
     // This must be the final send buffer; intermediate buffers cannot be freed.
     //
     CXPLAT_DATAPATH_PROC_CONTEXT* DatapathProc = SendData->Owner;
+#ifdef DEBUG
     uint8_t* TailBuffer = SendData->Buffers[SendData->BufferCount - 1].Buffer;
```

Comment:
I'm wondering how important this is to assert at all. Thoughts?

Reply:
I think its good to have, to know we are not passing in the incorrect buffer to free. But realistically I think there are unlikely to be bugs here.

Reply:
Ok.